Client/Frontend/UserContent/UserScripts/CookieControl.js

```diff
@@ -39,3 +39,13 @@ if (Object.getOwnPropertyDescriptor(window, 'sessionStorage')) {
         },
     });
 }
+
+//Access to caches should be denied when user has blocked all Cookies.
+if (Object.getOwnPropertyDescriptor(window, 'caches')) {
+    Object.defineProperty(window, 'caches', {
```

Comment:
this looks ok to me insofar as it's how the other ones work, but is it not possible for a page to bypass this by redefining `Object.defineProperty` before this is called?

Reply:
Old comment no longer applies